sdks/python/tests/e2e/test_tracing.py

```diff
@@ -1188,3 +1188,64 @@ def test_opik_client__update_span_with_attachments__original_fields_preserved_bu
         data_sizes=data_sizes,
         timeout=30,
     )
+
+
+def test_opik_client__update_trace__happy_flow(opik_client: opik.Opik):
```

Comment:
This test in theory should pass even if under the hood we upsert the new trace with the arguments passed because backend will overwrite the previous value.

Let's change it a little bit to only update **some** of the fields, and then in the end make sure that the fields that we haven't updated are still the same, and the updated ones have new value.